Source/Public/Publish-Bicep.ps1

```diff
@@ -33,10 +49,23 @@ function Publish-Bicep {
             Throw $_  
         }
 
+        # Set target for publish
+        if (-not $Target) {
+            if ($Registry -like '*azurecr.io*') {
```

Comment:
for clarity, remove the trailing star. We don't want a registry calling the police. (`callthecops.azurecr.iooooiiiiooooiiiiooo` is valid according to this pattern)